components/services/battery.c

```diff
@@ -43,8 +43,8 @@ static struct {
 /****************************************************************************************
  * 
  */
-int battery_value_svc(void) {
-	return battery.avg;
+double battery_value_svc(void) {
```

Comment:
can we use float as much as possible? esp32 has no support for double

Reply:
I can,  this call:

https://github.com/sle118/squeezelite-esp32/blob/29a74b9ca8e629eb2b2214d7e9b4ba91436b0c0d/components/wifi-manager/wifi_manager.c#L459

wants a double, so that would result in two conversions.  If you are ok with that, I'll change it.

Reply:
I mean the battery_value_svc proto was boviously a mistake on my side, it should return a float. Conversion to double will happen in the cJSON